iree/compiler/Conversion/Common/Transforms.cpp

```diff
@@ -443,5 +443,164 @@ LogicalResult tileAndFuseLinalgBufferOps(
   return success();
 }
 
+static bool isDivisible(Value v, int64_t dividend);
+
+/// Return true if we can prove that affineMinOp result is positive and
+/// divisible by the given |dividend|. This is true if all the the results of
+/// the associated affine map are positive and divisible by |dividend|.
+/// This speciically look for the following pattern:
+/// ```
+/// scf.for %iv = %lb to %ub step %step
+///   %affine.min affine_map<(d0, d1) -> (N, d0 - d1)>(%ub, %iv)
+/// ```
+static bool affineMinOpDivisible(AffineMinOp minOp, int64_t dividend) {
```

Comment:
Nit about name: This sounds like the affine_min is divisible by tile size :) . If you cant think of anything better, I cant either.

Reply:
Well it does mean the result of the affine_min is divisible by tile size. We check that both side of the min are divisible to prove that the result is divisible. If it wasn't we couldn't do the transformation. Maybe I misunderstood your comment.

Reply:
Its a minor nit. I already approved. My read was that "affine min is divisible" which to me is a strange name. I didnt parse it the same as "result of affine min is divisible".
Like I said, very minor issue. You can ignore it if this makes no sense to you.